pkg/provision/ec2.go

```diff
@@ -33,7 +33,11 @@ func (p *EC2Provisioner) Provision(host BasicHost) (*ProvisionedHost, error) {
 		return nil, err
 	}
 
-	groupID, name, err := p.securityGroup(host.Additional["inlets-port"])
+	port, err := strconv.Atoi(host.Additional["inlets-port"])
```

Comment:
This might vary with inlets-pro, the port range is decided by the client

Reply:
So for pro, we may want to open all ports.

Reply:
OK, I will update.

Reply:
I have added all ports if running pro.